cli/main.c

```diff
@@ -449,9 +476,9 @@ static int bw(int argc, char **argv)
 		DEVICE_OPTION,
 		{"time", 't', "NUM", CFG_POSITIVE, &cfg.meas_time,
 		  required_argument,
-		 "measurement time, in seconds"},
+		 "measurement time in seconds"},
 		{"verbose", 'v', "", CFG_NONE, &cfg.verbose, no_argument,
-		 "print posted, non-posted and completion results"},
+		 "print posted, non-posted, and completion results"},
```

Comment:
I don't think the oxford comma adds value here.

Reply:
Sure, I'll remove it.